docs/content/example_gallery.rst

```diff
@@ -19,15 +19,15 @@ The following image demonstrates that enhancer regions are typically
 small stretches of highly accessible chromatin (more information on
 enhancers can be found, for example,
 `here <http://dx.doi.org/doi:10.1038/nature07829>`__). In the heatmap,
-yellow and blue color tiles indicate large numbers of reads that were
-sequenced (which is indicative of open chromatin), black spots indicate
+yellow and blue tiles indicate a large numbers of reads that were
+sequenced (indicative of open chromatin) and black spots indicate
 missing data points. An appropriate labeling of the y-axis was
 neglected.
 
 **Fast Facts:**
 
 * `computeMatrix` mode: reference-point
-* *regions file*: BED file with typical enhancer regions from `Whyte et al., 2013 <http://dx.doi.org/10.1016/j.cell.2013.03.035>`__ (download `here <https://raw.github.com/fidelram/deepTools/master/gallery/Whyte_TypicalEnhancers_ESC.bed>`__)
+* *regions file*: BED file with typical enhancer regions from `Whyte et al., 2013 <http://dx.doi.org/10.1016/j.cell.2013.03.035>`__ (download `here <https://raw.github.com/fidelram/deepTools/release-1.6/gallery/Whyte_TypicalEnhancers_ESC.bed>`__)
```

Comment:
This reminds me ... when do we plan to do the merge? This should be changed back at time of merging.